src/components/Publish/_utils.ts

```diff
@@ -248,17 +248,6 @@ export async function createTokensAndPricing(
         poolParams
       )
 
-      // the spender in this case is the erc721Factory because we are delegating
-      const txApprove = await approve(
```

Comment:
just double-checking, are we sure we don't need an approve transaction here before creating the pool?

Reply:
sorry, it has been restore. I probably remove it while testing